Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android) // not available
```

Comment:
> Can you tell me the syntax to specify a NSTask unavailable for android please.

I think this means adding Android to [the list of platforms that the Swift compiler knows about](https://github.com/apple/swift/blob/bfc7924995d4de543e7b685faf49e8d25d563ce1/include/swift/AST/PlatformKinds.def). Adding that (and maybe some other tweaks) should allow us to write `@available(Android, unavailable)`. Does that sound right to you, @parkera?